### PR TITLE
Add split view sample

### PR DESCRIPTION
**Pull request by franek-scandit**

### CODE REVIEW

File: Native/iOSSplitViewSample/iOSAimAndScanSample/SplitViewResultTableViewController.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        {
+            base.ViewDidLoad();
+            splitViewResultsTableView.Source = new SplitViewResultTableSource(splitViewResults);
+            splitViewResultsTableView.ReloadData();
+        }
+
+        public void AddResult(SplitViewResult result)
+        {
+            splitViewResults.Insert(0, result);
+            splitViewResultsTableView.Source = new SplitViewResultTableSource(splitViewResults);
```
Comment (lucatorella) [inline Native/iOSSplitViewSample/iOSAimAndScanSample/SplitViewResultTableViewController.cs]: is there a reason for creating a new `SplitViewResultTableSource` every time?

File: Native/iOSSplitViewSample/iOSAimAndScanSample/ViewController.cs [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+                }
+            }
+        }
+
+        // Scanner timeOut in seconds.
+        private double timeOut = 10.0;
+        // Scan View Height in percents relative to ViewController's height.
+        private double scanViewHeight = 50.0;
+        // Determines if the beep sound will be played when the code will be found.
+        private bool beepEnablged = true;
```
Comment (lucatorella) [inline Native/iOSSplitViewSample/iOSAimAndScanSample/ViewController.cs]: typo

File: Native/iOSSplitViewSample/iOSAimAndScanSample/ViewController.cs [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+                        {
+                            continue;
+                        }
+                    }
+                });
+            }
+        }
+
+    }
+}
```
Comment (lucatorella) [inline Native/iOSSplitViewSample/iOSAimAndScanSample/ViewController.cs]: empty line missing at the end of the file

Comment (franek-scandit) [reply Native/iOSSplitViewSample/iOSAimAndScanSample/ViewController.cs]: Fixed

**Comment by franek-scandit** [comment]
@lucatorella I fixed all issues you commented about.

**Pull request merged**